.storybook/main.ts

```diff
@@ -2,6 +2,7 @@ module.exports = {
   stories: ["../src/**/*.stories.@(js|jsx|ts|tsx)"],
   addons: [
     "storybook-addon-preview",
+    "@storybook/addon-docs",
```

Comment:
To suppress warning in Storybook cli logs.
addon-docs must be loaded before controls